src/Examples/UnitModels/Operations/feed_testing.ipynb

```diff
@@ -0,0 +1,376 @@
+     ]
+    }
+   },
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tutorial: Feed Unit Model with Ideal Property Package\n",
+    "\n",
+    "\n",
+    "![](feed.svg)\n",
+    "![feed.svg](attachment:feed.svg)\n",
+    "\n",
+    "## Learning Outcomes\n",
+    "\n",
+    "- Demonstrate use of the Feed unit model in IDAES\n",
+    "\n",
+    "\n",
+    "## Problem Statement\n",
+    "\n",
+    "In this tutorial, we will be feeding a mixed stream of liquid benzene and liquid toluene. This unit model has no inlets and only one outlet. The outlet conditions are as follows:\n",
+    "\n",
+    "Flow Rate = 100 mol/s\n",
+    "\n",
+    "Mole Fraction (Benzene) = 0.6\n",
+    "\n",
+    "Mole Fraction (Toluene) = 0.4\n",
+    "\n",
+    "Temperature = 298.15 K\n",
+    "\n",
+    "Pressure = 101325 Pa\n",
+    "\n",
+    "For more details, please refer to the IDAES documentation: https://idaes-pse.readthedocs.io/en/stable"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setting up the problem in IDAES"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the following cell, we will be importing the necessary components from Pyomo and IDAES."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import objects from pyomo package \n",
+    "from pyomo.environ import ConcreteModel, SolverFactory, value\n",
+    "\n",
+    "# Import the main FlowsheetBlock from IDAES. The flowsheet block will contain the unit model\n",
+    "from idaes.core import FlowsheetBlock\n",
+    "\n",
+    "# Import the feed unit model\n",
+    "from idaes.models.unit_models import Feed\n",
+    "\n",
+    "# Import idaes logger to set output levels\n",
+    "import idaes.logger as idaeslog\n",
+    "\n",
+    "# Import the BTX_ideal property package to create a properties block for the flowsheet\n",
+    "from idaes.models.properties.activity_coeff_models import BTX_activity_coeff_VLE\n",
```

Comment:
It would be better to use the modular properties here if possible. The activity coefficient package has not been well maintained, and may need to be changed or deprecated in the future. Plus, it would be good to show the modular properties more.